lib/rubocop/git/commit_file.rb

```diff
@@ -29,7 +29,7 @@ def removed?
   end
 
   def ruby?
-    filename.match(/.*\.(rb|rake)|Gemfile|Rakefile$/)
+    filename.match(/.*\.(rb|rake)|Gemfile$|Rakefile$/)
```

Comment:
Since we already broke this once, can we write a test for it? It should be easy to exhaust all the cases. We also have many `Whateverfile` in Rubyland, we can also do a bit more robust check here like:

```ruby

RUBY_EXT = %w(rb rake)
RUBY_FILES = %w(Gemfile Rakefile Procfile Guardfile)

def ruby?
  filename.match(/.*\.#{Regexp.union(RUBY_EXT}|^(#{Regexp.union(RUBY_FILES})$/)
end
```

Or maybe even drop the regexp for the `Whateverfiles` and do a manual comparison. Anyway, I'm sure we can figure a robust way to check for it.